src/app/services/user-management/user.service.ts

```diff
@@ -50,60 +46,56 @@ export class UserService {
     });
   }
 
-  createUser(username: string, password: string): Observable<CreateUserStatus> {
-    return new Observable<CreateUserStatus>((observer) => {
-      this.httpClient
-        .post(this.userEndpoint, { username: username, password: password })
-        .subscribe(
-          (body) => {
-            observer.next(CreateUserStatus.CREATE_OK);
-          },
-          (err: HttpErrorResponse) => {
-            console.error(err);
-            if (err.status === 409) {
-              observer.error(CreateUserStatus.CREATE_ERR_EXISTS);
-            } else {
-              observer.error(CreateUserStatus.CREATE_ERR_SERVER);
-            }
-          }
-        );
-    });
+  createUser(username: string, password: string): Observable<UserStatusCode> {
+    return this.httpClient
+      .post(this.userEndpoint, { username: username, password: password })
+      .pipe(
+        map((x) => {
+          return UserStatusCode.REQUEST_OK;
+        }),
+        catchError((err: HttpErrorResponse) => {
+          return ErrorObservable.create({
+            code: err.status,
+            message: 'stub',
+          });
+        })
+      );
   }
 
   // TODO(egeldenhuys): Update using model
   updateUser(username: string, password: string) {
-    return new Observable<UpdateUserStatus>((observer) => {
+    return new Observable<UserStatusCode>((observer) => {
       this.httpClient
         .put(this.userEndpoint + '/' + username, { password: password })
         .subscribe(
           (body) => {
-            observer.next(UpdateUserStatus.UPDATE_OK);
+            observer.next(UserStatusCode.REQUEST_OK);
           },
           (err: HttpErrorResponse) => {
             console.error(err);
             if (err.status === 404) {
-              observer.error(UpdateUserStatus.UPDATE_ERR_NOT_FOUND);
+              observer.error(UserStatusCode.REQUEST_ERR_NOT_FOUND);
```

Comment:
And in this case a `UserStatusCode` is returned on error?

Reply:
Whoops, guess we missed those; Will resolve either tonight or tomorrow.